elide-core/src/test/java/com/yahoo/elide/core/LifeCycleTest.java

```diff
@@ -115,22 +127,21 @@ public TestEntityDictionary(Map<String, Class<? extends Check>> checks) {
         dictionary.bindEntity(Author.class);
         dictionary.bindEntity(Publisher.class);
         dictionary.bindEntity(Editor.class);
-        dictionary.bindTrigger(Book.class, OnCreatePostCommit.class, callback);
-        dictionary.bindTrigger(Book.class, OnCreatePreCommit.class, callback);
-        dictionary.bindTrigger(Book.class, OnCreatePreSecurity.class, callback);
-        dictionary.bindTrigger(Book.class, OnReadPostCommit.class, callback);
-        dictionary.bindTrigger(Book.class, OnReadPreCommit.class, callback);
-        dictionary.bindTrigger(Book.class, OnReadPreSecurity.class, callback);
-        dictionary.bindTrigger(Book.class, OnDeletePostCommit.class, callback);
-        dictionary.bindTrigger(Book.class, OnDeletePreCommit.class, callback);
-        dictionary.bindTrigger(Book.class, OnDeletePreSecurity.class, callback);
-        dictionary.bindTrigger(Book.class, OnUpdatePostCommit.class, "title", callback);
-        dictionary.bindTrigger(Book.class, OnUpdatePreCommit.class, "title", callback);
-        dictionary.bindTrigger(Book.class, OnUpdatePreSecurity.class, "title", callback);
+        ImmutableList.of(
+                OnCreatePostCommit.class, OnCreatePreCommit.class, OnCreatePreSecurity.class,
+                OnReadPostCommit.class, OnReadPreCommit.class, OnReadPreSecurity.class,
+                OnDeletePostCommit.class, OnDeletePreCommit.class, OnDeletePreSecurity.class)
+                .stream().forEach(cls -> dictionary.bindTrigger(Book.class, cls, callback));
+        ImmutableList.of(
+                OnUpdatePostCommit.class, OnUpdatePreCommit.class, OnUpdatePreSecurity.class)
+                .stream().forEach(cls -> dictionary.bindTrigger(Book.class, cls, "title", callback));
         dictionary.bindTrigger(Book.class, OnUpdatePreCommit.class, onUpdateDeferredCallback, true);
         dictionary.bindTrigger(Book.class, OnUpdatePreSecurity.class, onUpdateImmediateCallback, true);
         dictionary.bindTrigger(Book.class, OnUpdatePostCommit.class, onUpdatePostCommitCallback, true);
         dictionary.bindTrigger(Author.class, OnUpdatePostCommit.class, onUpdatePostCommitAuthor, true);
+        // enable trace
+        Logger rootLogger = (Logger) LoggerFactory.getILoggerFactory().getLogger(Logger.ROOT_LOGGER_NAME);
+        rootLogger.setLevel(Level.TRACE);
```

Comment:
Do we want to leave this log level on?

Reply:
Yes, otherwise we are not testing logging works without throwing exceptions.